de.fu_berlin.inf.dpp.server/src/de/fu_berlin/inf/dpp/server/session/NegotiationHandler.java

```diff
@@ -93,52 +98,53 @@ public void run() {
 
     @Override
     public void handleOutgoingProjectNegotiation(
-        final AbstractOutgoingProjectNegotiation negotiation) {
+        final AbstractOutgoingReferencePointNegotiation negotiation) {
 
         projectExecutor.execute(new Runnable() {
             @Override
             public void run() {
-                ProjectNegotiation.Status status;
+                ReferencePointNegotiation.Status status;
                 if (ServerConfig.isInteractive()) {
-                    status = negotiation
-                        .run(new ConsoleProgressIndicator(System.out));
+                    status = negotiation.run(new ConsoleProgressIndicator(
+                        System.out));
                 } else {
-                    status = negotiation
-                        .run(new NullProgressMonitor());
+                    status = negotiation.run(new NullProgressMonitor());
                 }
-                
+
                 switch (status) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Complete cases by adding the missing enum constants or add a default case to this switch. ([squid:SwitchLastCaseIsDefaultCheck](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=273&issues=AWcmJq5RkEeD8QRkTSvr&open=AWcmJq5RkEeD8QRkTSvr)

[ISSUE_KEY]: AWcmJq5RkEeD8QRkTSvr